lib/Jsrt/Jsrt.cpp

```diff
@@ -1342,7 +1342,7 @@ CHAKRA_API JsInstanceOf(_In_ JsValueRef object, _In_ JsValueRef constructor, _Ou
 
 CHAKRA_API JsGetExtensionAllowed(_In_ JsValueRef object, _Out_ bool *value)
```

Comment:
>JsGetExtensionAllowed [](start = 11, length = 21)

For most of the APIs below can call into user code if the object is proxy and you have handler for respective methods. See [https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy)